src/Math/babylon.math.ts

```diff
@@ -1083,16 +1083,17 @@
             var nbRevert = 0;
             var cross: Vector3;
             var dot = 0.0;
+            var epsilon = 0.001;
```

Comment:
modified in Path3D and in RotationFromAxis : 2 different PR


Le 01/09/2015 16:25, Raanan Weber a écrit :
>
> In src/Math/babylon.math.ts 
> <https://github.com/BabylonJS/Babylon.js/pull/678#discussion_r38424075>:
>
> > @@ -1083,16 +1083,17 @@
> >              var nbRevert = 0;
> >              var cross: Vector3;
> >              var dot = 0.0;
> > +            var epsilon = 0.001;
>
> Why don't you use Engine.epsilon?
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/BabylonJS/Babylon.js/pull/678/files#r38424075>.
>